packages/orbit-components/src/Dialog/index.js

```diff
@@ -154,7 +155,7 @@ const Dialog = ({
 
   const dialogID = React.useMemo(() => randomID("dialog"), []);
 
-  return (
+  return insidePortal ? (
```

Comment:
So "insidePortal" is short for "render this inside portal"? Initially I thought that it meant "it's already inside portal, so don't render another one". Instead I recommend alternatives like `renderInPortal`, `withPortal` or `hasPortal`.